app/assets/javascripts/common/reports.js

```diff
@@ -676,7 +682,9 @@ Reports = function ({
 
     const decemberKeys = dateKeysArray.filter((item) => item.includes("Dec"));
     const isLastDateKeysArrayMonthDec =
-      monthIndexFromDateString(dateKeysArray[dateKeysArray.length - 1]) === 11;
+      monthIndexFromChartLabelDateString(
+        dateKeysArray[dateKeysArray.length - 1]
+      ) === 11;
     if (isLastDateKeysArrayMonthDec) {
```

Comment:
Nitpick: We could rename  'isLastDateKeysArrayMonthDec' to 'isEndMonthOfYear' or 'isLastMonthOfYear'. This makes the function a little more clear.